flow/activities/flowable_core.go

```diff
@@ -276,7 +276,8 @@ func syncCore[TPull connectors.CDCPullConnectorCore, TSync connectors.CDCSyncCon
 	syncStartTime := time.Now()
 	if err := errGroup.Wait(); err != nil {
 		// don't log flow error for "replState changed" and "slot is already active"
-		if !(temporal.IsApplicationError(err) || shared.IsSQLStateError(err, pgerrcode.ObjectInUse)) {
+		var applicationError *temporal.ApplicationError
+		if !((errors.As(err, &applicationError) && applicationError.Type() == "desync") || shared.IsSQLStateError(err, pgerrcode.ObjectInUse)) {
```

Comment:
shouldn't it be disconnect? ref https://github.com/PeerDB-io/peerdb/blob/0eac5fa55b7346d12213abd954778f3778db38ec/flow/connectors/postgres/postgres.go#L385-L395

Reply:
this is checking for what to ignore, we want to ignore `desync` not `disconnect`